deploy/post_deploy_testing/cypress/integration/03a_browse_views_basic.js

```diff
@@ -5,40 +5,42 @@ describe('Browse Views - Basic Tests', function () {
     context('Navigation and Redirection', function(){
 
         it('If start from home page, clicking on Browse nav menu item gets us to Browse page.', function(){
-
+            const dataNavBarItemSelectorStr = '#top-nav div.navbar-collapse .navbar-nav a.id-data-menu-item';
             cy.visit('/');
+            cy.get(dataNavBarItemSelectorStr).click().wait(500).then(()=>{
+                
+            });
 
             cy.get(navBrowseBtnSelector).click().then(()=>{
                 cy.get('#page-title-container .page-title span.title').should('have.text', 'Data Browser');
             });
 
         });
 
-        it('Only shows award.project=4DN results & "Include External Data" is off', function(){
-
-            cy.location('search').should('include', 'award.project=4DN');
-            cy.get('#stats .browse-base-state-toggle-container input[type="checkbox"]').should('not.be.checked');
+        it('Only shows award.project=4DN results & "Include External Data" is off', function(){           
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4521913)